### DISCUSSION TITLE
My translations made through Weblate are not recognized by GitHub as mine

### DISCUSSION THREAD

**Discussion by DursunCanPoyraz** [rob006-software/flarum-translations#570] [General]
How can I connect my Github account with my weblate membership?
I cannot add a new e-mail address. The site has failed.
It would be nice to have my GitHub profile visible in Commits.

![image](https://user-images.githubusercontent.com/74768208/102925906-400fc500-44a5-11eb-9e3a-7366e0da9092.png)

GitHub Info:
- Username: DursunCanPoyraz
- Mail: info@dursuncanpoyraz.com.tr

Weblate Info:
- Username: DursunCanPoyraz
- Mail: 61northeaster@gmail.com

**Comment by DursunCanPoyraz** [comment]
Mail problem solved.

![image](https://user-images.githubusercontent.com/74768208/102926918-f45e1b00-44a6-11eb-952d-2b4ddadc8798.png)

**Comment by rob006** [comment]
It is better to add your Weblate email to your GitHub account at https://github.com/settings/emails (you can have multiple emails connected with single GitHub account) - it should also work for old commits.

**Comment by DursunCanPoyraz** [comment]
![image](https://user-images.githubusercontent.com/74768208/102927454-173cff00-44a8-11eb-8ca6-e22cadf744bd.png)

Solved.

**Comment by rob006** [comment]
I'll mention this at https://github.com/rob006-software/flarum-translations/wiki

**Comment by rob006** [comment]
https://github.com/rob006-software/flarum-translations/wiki#why-my-translations-made-through-weblate-are-not-recognized-by-github-as-mine
